scss/_close.scss

```
@@ -7,11 +7,15 @@
   text-shadow: $close-text-shadow;
   opacity: .5;
 
+  // Override <a>'s hover style
+  &:hover {
```

We should use the hover mixin here:
```suggestion
  @include hover {
```